src/components/modes/student/StudentView.js

```diff
@@ -1,12 +1,7 @@
 import React from 'react';
 import { withTranslation } from 'react-i18next';
-import { withStyles } from '@material-ui/core/styles';
 import Canvas from '../../main/Canvas';
 
-const styles = () => ({});
-
 const StudentView = () => <Canvas />;
 
-const StyledComponent = withStyles(styles)(StudentView);
-
-export default withTranslation()(StyledComponent);
+export default withTranslation()(StudentView);
```

Comment:
I think if you use `useTranslation` hook inside all your functional components and you don't have any class components you can remove this, as it is useful to pass the `t` and `i18n` instances as props to the component.